src/transformers/onnx/features.py

```diff
@@ -192,6 +198,16 @@ class FeaturesManager:
             "question-answering",
             onnx_config_cls=CamembertOnnxConfig,
         ),
+        "mobilebert": supported_features_mapping(
```

Comment:
Please put this entry in alphabetical order after rebasing on `main`

Reply:
@lewtun Done! :)